src/objects/list/list.scss

```diff
@@ -0,0 +1,56 @@
+@use '../../design-tokens/colors.yml';
+@use '../../design-tokens/sizes.yml';
+
+/**
+ * Override browser defaults. Note that this means it's very important to
+ * include `[role="list"]` to prevent disruption of list navigation in Safari
+ * VoiceOver.
+ *
+ * @see https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html
+ */
+
+.o-list {
+  list-style: none;
+  padding-left: 0;
+}
+
+/**
+ * Modifier: Inline
+ *
+ * By default, this uses negative margins on the sides to account for extra
+ * margin between child elements. This allows child elements to reach the edges
+ * of their parent while maintaining consistent gaps between.
+ *
+ * This would be a _lot_ more elegant if it used the Flexbox-compatible `gap`
+ * property. But unfortunately there is no way to test for that without false
+ * positives as of this writing.
```

Comment:
Is my understanding correct, that at this time there is no way to wrap a simplified version of this, using `gap` in a `@supports` media query?

Reply:
Yep! See the `@see` URLs for more information.